client/idm/console/src/main/resources/org/apache/syncope/client/console/wizards/resources/AbstractConnConfPanel.html

```diff
@@ -21,7 +21,7 @@
     <div id="emptyPlaceholder"/>
     <span wicket:id="connectorPropertiesContainer">
       <div class="pull-right">
-        <a style="position: fixed; top: 65px; right:60px;" wicket:id="check"  href="#">
+        <a style="position: fixed; top: 95px; right:420px;" wicket:id="check"  href="#">
```

Comment:
Here it would be better to use `position: absolute; top: 0; right: 20px;`, because the rule with `fixed` seems to be not responsive when changing device or just resizing the window.

Reply:
...and a `z-index: 1` would be good to have in order not to hide the icon behind other fields!

Reply:
after a deeper check this solution is not good, since the `Override?` flag bothers the check icon.
A totally better solution is to edit that html file as following:
```html
<html xmlns="http://www.w3.org/1999/xhtml" xmlns:wicket="http://wicket.apache.org">
  <wicket:panel>
    <div id="emptyPlaceholder"/>
    <span wicket:id="connectorPropertiesContainer">
      <a class="conn-check-wrapper" wicket:id="check"  href="#">
        <i class="fa fa-heartbeat fa-2x"></i>
      </a>
<!-- ... -->
```

and the `syncopeUi.scss` file needs to include:
```scss
.scrollable-tab-content {
  padding-top: 30px;
  /* other already existing rules */
}
.conn-check-wrapper {
  position: absolute;
  top: 0;
  right: 25px;
  z-index: 1;
}
```